job.go

```diff
@@ -122,42 +121,18 @@ type JobRow struct {
 	// metadata is a field that'll eventually be used to store arbitrary data on
 	// a job for flexible use and use with plugins. It's currently unexported
 	// until we get a chance to more fully flesh out this feature.
-	metadata []byte
+	// metadata []byte
 }
 
-// WARNING!!!!!
-//
-// !!! When updating this function, the equivalent in `./rivertest/rivertest.go`
-// must also be updated!!!
-//
-// This is obviously not ideal, but since JobRow is at the top-level package,
-// there's no way to put a helper in a shared package that can produce one,
-// which is why we have this copy/pasta. There are some potential alternatives
-// to this, but none of them are great.
-func jobRowFromInternal(internal *dbsqlc.RiverJob) *JobRow {
```

Comment:
This moves into `dbsqlc`, and like #36, we can remove the duplicate that's in `rivertest`.